backend/controllers/webhook.js

```diff
@@ -42,6 +43,15 @@ router.post('/', (req, res) => {
       console.log(`Meeting ${meetingName} started at ${timeStarted}`);
       return res.status(201).json({ message: 'Meeting has started.' });
     }
+    case WEBHOOK_PARTICIPANT_LEFT: {
+      const participantId = req.body.payload.object.participant.id;
```

Comment:
Can we use `participant_user_id` instead of `id` here? I saw this note in the zoom docs for `participant_left`:

> Note: Use the participant_user_id value instead of this value. We will remove this response in a future release.